dd-sdk-android/src/test/kotlin/com/datadog/android/core/configuration/ConfigurationBuilderTest.kt

```diff
@@ -1234,4 +1241,26 @@ internal class ConfigurationBuilderTest {
             Configuration.Feature.InternalLogs(clientToken, url, emptyList())
         )
     }
+
+    @Test
+    fun `𝕄 build config with Span eventMapper 𝕎 setSpanEventMapper() and build()`() {
+        // Given
+        val eventMapper: SpanEventMapper = mock()
+
+        // When
+        val config = testedBuilder
+            .setSpanEventMapper(eventMapper)
+            .build()
+
+        // Then
+        assertThat(config.coreConfig).isEqualTo(Configuration.DEFAULT_CORE_CONFIG)
+        assertThat(config.logsConfig).isEqualTo(Configuration.DEFAULT_LOGS_CONFIG)
+        assertThat(config.rumConfig).isEqualTo(Configuration.DEFAULT_RUM_CONFIG)
+        assertThat(config.tracesConfig!!).isEqualTo(
```

Comment:
is `!!` needed here?

Reply:
Hmmm...I am not sure how this ended up here :)